askama_shared/src/lib.rs

```diff
@@ -37,9 +37,21 @@ pub trait Template: fmt::Display {
         Ok(buf)
     }
 
-    /// Renders the template to the given `writer` buffer
+    /// Helper method which allocates a new `Vec<u8>` and renders into it
+    #[inline]
+    fn render_bytes(&self) -> Result<Vec<u8>> {
```

Comment:
This doesn't meet my cost/benefit bar, conversions with `String::into_bytes()` should be easy.

Reply:
Removed.